template/registry_datetime.go

```diff
@@ -39,12 +39,19 @@ func parseDatetimeInput(input any) (time.Time, error) {
 		return time.Time{}, fmt.Errorf("input is required: %w", ErrOperandMismatch)
 	}
 
-	v, ok := input.(time.Time)
-	if !ok {
-		return time.Time{}, fmt.Errorf("unsupported type %T: %w", input, ErrOperandMismatch)
-	}
+	switch v := input.(type) {
+	default:
+		return time.Time{}, fmt.Errorf("unsupported datetime type %T: %w", input, ErrOperandMismatch)
+	case string:
+		t, err := time.Parse(time.RFC3339[:len(v)], v)
```

Comment:
Interesting approach with dynamic format layout :+1:

Reply:
Quick and dirty. It does not conform fully to ISO 8601 formatting. E.g. `"2006-01-02T15:0` is not valid ISO 8601. I'll fix it.

Reply:
Not planning to fix it now. However, I added the comment.